ably/paginated_resource_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("PaginatedResource", func() {
 		})
 
 		It("returns a string pointing to the new path based on the given path", func() {
-			newPath, err := ably.BuildPath(&paginatedResource, "/path/to/resource?hello", "./newresource?world")
+			newPath, err := paginatedResource.BuildPath("/path/to/resource?hello", "./newresource?world")
```

Comment:
This is my fault because in Ruby I named the class `PaginatedResource`, but in Ably-js and Ably-Ruby it's called `PaginatedResult`.  Sorry, entirely my fault I expect.  See https://github.com/ably/ably-ruby/issues/40